valhalla/sif/costfactory.h

```diff
@@ -101,6 +101,7 @@ class CostFactory {
 
   mode_costing_t CreateModeCosting(const Options& options, TravelMode& mode) {
     mode_costing_t mode_costing;
+    mode = TravelMode::kMaxTravelMode;
```

Comment:
some of the tests were all of a sudden throwing a "mode might be uninitialized" warning. it's technically possible I guess, in case we'd happen to add another costing type which isn't represented in `kCostingTypeMapping`. makes me think of https://github.com/valhalla/valhalla/pull/4492, in case you're not yet aware of it @ianthetechie :)

Reply:
heh ;) Thanks for the heads up @nilsnolde. This is vaguely familiar but making a note to review when I get back to finalizing that PR.